.github/workflows/scanner-update-nvd.yaml

```diff
@@ -0,0 +1,38 @@
+name: Fetch and Update NVD CVEs
+
+on:
+  push:
+    branches:
+      - yli3/**
+  schedule:
+    - cron: '0 0 * * *'  # this runs every day at midnight
+
+jobs:
+  fetch-cves:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Authenticate with Google Cloud
+      uses: google-github-actions/auth@v1
+      with:
+        credentials_json: ${{ secrets.GOOGLE_SA_CIRCLECI_SCANNER }}
+
+    - name: Set up Cloud SDK
+      uses: google-github-actions/setup-gcloud@v1
+
+    - name: Fetch Critical CVEs
+      env:
+        API_KEY: ${{ secrets.NVD_API_KEY }}
+      run: ./.github/workflows/scripts/scanner-fetch-nvd.sh
```

Comment:
I know it was in this directory before, but is running the script from this directory not frowned upon? I figured putting all CI-related scripts into `scripts/ci` was preferred, but I could be wrong. I'll defer this to @BradLugo or @tommartensen

Reply:
`.github/workflows/scripts/` primarily exists for the scripts used in the release automation, which we call using `.github/workflows/scripts/common.sh`. 

The way I understand the `scanner-update-nvd.yaml`, it is not CI in the sense of it building or testing something or running for every commit, rather a scheduled job. In that way, `.github/workflows/scripts` may be fine.